magento/adobe-stock-integration
# [MFTF] Cover test scenarios #1046, #1047

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1046: Cover creating new folder functionality in Media Gallery
2. magento/adobe-stock-integration#1047: Cover deleting folder functionality in Media Gallery

### Manual testing scenarios (*)

- See https://github.com/magento/adobe-stock-integration/issues/1046
- See https://github.com/magento/adobe-stock-integration/issues/1047
